app/src/main/java/com/elavon/converge/processor/ConvergeClient.java

```diff
@@ -88,7 +88,9 @@ public void onFailure(final Call call, final IOException e) {
                 callback.onFailure(e);
             }
         };
-        client.newCall(request).enqueue(cb);
+        if(request != null) {
+            client.newCall(request).enqueue(cb);
+        }
```

Comment:
this is bad right - the caller would never get any call back if the request is null for any reason - call the callback.onFailure() when it's null.

Reply:
done